spec/factories/file_attachment.rb

```diff
@@ -0,0 +1,20 @@
+require './lib/pivotal-tracker/file_attachment'
+
+FactoryGirl.define do
+  factory :file_attachment, class: PivotalTracker::FileAttachment do
+    sequence(:id)
+    filename      'filename'
+    created_at    DateTime.now
+    uploader_id   1
+    thumbnailable true
+    height        1
+    width         1
+    size          1
+    download_url  { Faker::Internet.url }
+    content_type  'content_type'
+    uploaded      true
+    big_url       { Faker::Internet.url }
+    thumbnail_url { Faker::Internet.url }
+    kind          'file_attachment'
+  end
```

Comment:
@jasonnoble So I changed all of these to be strings, but the docs say that the strings are objects, but it doesn't seem reasonable to store objects in these factories. 

From the docs:

string 
 —  The type of this object: file_attachment. This field is read only.

source:
https://www.pivotaltracker.com/help/api/rest/v5#file_attachment_resource